### ISSUE TITLE
« "boolean-negation": false » only works when running from the package directory

### ISSUE THREAD

**Issue by jean-emmanuel** [yargs/yargs#1248]
When running a node app from a directory that doesn't contain the appropriate package.json, boolean negation is always enabled. Any chance this could be disabled programmatically ?
Thanks a lot for the lib :)

**Comment by jean-emmanuel** [comment]
Note: packaged node applications (eg electron apps) are likely to be run from an arbitrary directory that may not contain the app's `package.json`.

**Comment by jean-emmanuel** [comment]
Yargs relies on `process.cwd()` to find the package.json file, this method doesn't seem to work when the scripts are bundled with browserify. Here's my workaround:

```js
var _cwd = process.cwd
process.cwd = () => { return __dirname }

// require yargs, etc
process.cwd = _cwd
```

**Comment by bcoe** [comment]
@jean-emmanuel YES, we've been having an ongoing conversation in the yargs slack channel about making this something we can configure programmatically -- I think I was trying to be overly clever when I implemented this feature originally.

Want to rephrase this as a feature request, and let's implement this?

CC: @cspotcode

**Comment by jean-emmanuel** [comment]
Alright, PR on the way :)

**Comment by jean-emmanuel** [comment]
Yes, sorry for the delay, it works perfectly !
